src/core/cpuid.d

```diff
@@ -143,7 +143,7 @@ public:
     /// Is SSE3 supported?
     bool sse3()         {return (miscfeatures&SSE3_BIT)!=0;}
     /// Is SSSE3 supported?
-    bool ssse3()        {return (miscfeatures&SSSE3_BIT)!=0;}
```

Comment:
> why not just make miscfeature immutable and leave functions as is

Really good idea. You could make the storage a single immutable struct, then set the value from the shared static ctor based on calling some function.